Google.PowerShell/PubSub/GcpsTopic.cs

```diff
@@ -0,0 +1,234 @@
+    /// matching the topic names specified in -Topic and will throw an error for any topic that cannot be found.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsTopic</code>
+    ///   <para>This command retrieves all topics in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsTopic -Topic "topic1", "topic2" -Project "my-project"</code>
+    ///   <para>
+    ///   This command retrieves 2 topics ("topic1" and "topic2") in the project "my-project".
+    ///   </para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/pubsub/docs/publisher#list-topics-in-your-project)">
+    /// [Listing a Topic]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcpsTopic")]
+    public class GetGcpsTopic : GcpsCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to check for topics. If not set via PowerShell parameter processing, will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The names of the topics to be retrieved.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        [Alias("Name")]
+        public string[] Topic { get; set; }
+
+        protected override void ProcessRecord()
+        {
+            if (Topic != null && Topic.Length > 0)
+            {
+                foreach (string topicName in Topic)
+                {
+                    string formattedTopicName = PrefixProjectToTopic(topicName, Project);
+                    try
+                    {
+                        ProjectsResource.TopicsResource.GetRequest getRequest = Service.Projects.Topics.Get(formattedTopicName);
+                        WriteObject(getRequest.Execute());
+                    }
+                    catch (GoogleApiException ex) when (ex.HttpStatusCode == HttpStatusCode.NotFound)
+                    {
+                        ErrorRecord errorRecord = new ErrorRecord(
+                            new ItemNotFoundException($"Topic '{topicName}' does not exist in project '{Project}'."),
+                            "TopicNotFound",
+                            ErrorCategory.ObjectNotFound,
+                            topicName);
+                        WriteError(errorRecord);
+                    }
+                }
+            }
+            else
+            {
+                ProjectsResource.TopicsResource.ListRequest listRequest = Service.Projects.Topics.List($"projects/{Project}");
+                do
+                {
+                    ListTopicsResponse response = listRequest.Execute();
+
```

Comment:
Remove this line of whitespace? The code seems straight forward enough to not need it.